app/controllers/line_items_controller.rb

```diff
@@ -20,10 +23,12 @@ def bought_items
     current_order_cycle.items_bought_by_user(spree_current_user, current_distributor)
   end
 
-  # Override default which just redirects
-  # See Spree::BaseController and Spree::Core::ControllerHelpers::Auth
   def unauthorized
-    render text: '', status: 403
+    render nothing: true, status: 401 and return
+  end
+
+  def not_found
+    render nothing: true, status: 404 and return
   end
```

Comment:
if we are going to use `not_found` and `unouthorized` in others API controllers we should probably move them to some OFN base API controller.